code-challenges.test.js

```diff
@@ -32,15 +133,87 @@ const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
 const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
+describe("remainderArray", () => {
+  it("returns a an array of only the remainders of the number when divided by 3", () => {
+    expect(remainderArray(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+    expect(remainderArray(hodgepodge2)).toEqual([ 2, 1, -1 ])
+  })
+})
+
+//Test failed:  ReferenceError: remainderArray is not defined
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 passed, 2 total
+
+
 // b) Create the function that makes the test pass.
 
+//Pseudo:
+//Create a function called remainderArray.
+//Input: A mixed data array.
+//Iterate over array using .map
+//Check if each element is a number by using typeof
+//If element is a number, then use modulo by 3 to get the remainder
+//Use .filter to remove undefined values out of newArray 
+//Output: Returns an array of only the remainders of the number when divided by 3.
+
+const remainderArray = (array) => {
+  let newArray = []
```

Comment:
Logic looks great on this problem!  Since map and filter both return a new array, there is no need to create an empty array in the beginning.  You can either assign the mapped or filtered array on the same line, or even just return it without assigning a variable.